### ISSUE TITLE
mojaloop-simulator helm chart template is not handling booleans logical comparisons correctly against VALIDATE_INBOUND_JWS value config

### ISSUE THREAD

**Issue by mdebarros** [mojaloop/project#2083]
**Summary**:
When installing Mojaloop v12.0.0 and setting VALIDATE_INBOUND_JWS = "false" and pubKeyConfigMapName is not defined (or false) config values on the Mojaloop-simulator Helm chart <-- i.e. set `simulators.${each.value}.config.schemeAdapter.env.VALIDATE_INBOUND_JWS` to "false" as a string and not a boolean

The following error is displayed:
```
Error: template: mojaloop-simulator/templates/config-jws-public-keys.yaml:14:8: executing “mojaloop-simulator/templates/config-jws-public-keys.yaml” at <fail (printf “INBOUND_JWS enabled and jws public key not specified for ‘%s’. You need to specify %s.schemeAdapter.secrets.jws.publicKey or %s.schemeAdapter.secrets.jws.pubKeyConfigMapName.” $name $name $name)>: error calling fail: INBOUND_JWS enabled and jws public key not specified for ‘pm4mlsenderfsp’. You need to specify pm4mlsenderfsp.schemeAdapter.secrets.jws.publicKey or pm4mlsenderfsp.schemeAdapter.secrets.jws.pubKeyConfigMapName.
```

On line 12 of this template: https://github.com/mojaloop/helm/blob/master/mojaloop-simulator/templates/config-jws-public-keys.yaml#L12

You will notice that the if-statement is only comparing that the VALIDATE_INBOUND_JWS value is not "null" or false. So as long as the value is either a boolean `true` or is a string it will always be true.

**Severity**:
Low

**Priority**:
Low

**Expected Behavior**

The following if-statement should not be entered when https://github.com/mojaloop/helm/blob/master/mojaloop-simulator/templates/config-jws-public-keys.yaml#L12: 
1. When setting `simulators.${each.value}.config.schemeAdapter.env.VALIDATE_INBOUND_JWS` to a string such as "false" or "hello"
2. When setting `simulators.${each.value}.config.schemeAdapter.env.VALIDATE_INBOUND_JWS` to a boolean false

**Steps to Reproduce**
  1. Install Mojaloop v12.0.0 release with `simulators.${each.value}.config.schemeAdapter.env.VALIDATE_INBOUND_JWS` to "false" as a string and not a boolean, and `simulators.${each.value}.config.schemeAdapter.env.pubKeyConfigMapName` not set/defined

**Specifications**
  - Component (if known): Mojaloop Helm
  - Version: v12.0.0
  - Platform: Helm
  - Subsystem:
  - Type of testing: Manual
  - Bug found/raised by: @mdebarros @dfry 

**Notes**:

- Severity when opened: Low
- Priority when opened: Low

**Comment by mdebarros** [comment]
Fix applied in Helm 12.1.0 release -> https://github.com/mojaloop/helm/pull/436/commits/f423d4b5bb5495c5c5f73a8884043c297070a971

**Comment by elnyry-sam-k** [comment]
Fixed with https://github.com/mojaloop/helm/releases/tag/v13.0.0
